addons/sourcemod/scripting/mgemod.sp

```diff
@@ -455,25 +485,25 @@ public OnMapStart()
  * Repeating timers can be killed here.
  * Hooks are removed here.
  * -------------------------------------------------------------------------- */
-public OnMapEnd()
+public void OnMapEnd()
 {
-	g_hDBReconnectTimer = INVALID_HANDLE;
-	g_bNoStats = (GetConVarBool(gcvar_stats)) ? false : true;
-
+	g_hDBReconnectTimer = null;
+	g_bNoStats = gcvar_stats.BoolValue ? false : true;
+	
 	UnhookEvent("player_death", Event_PlayerDeath, EventHookMode_Pre);
 	UnhookEvent("player_spawn", Event_PlayerSpawn, EventHookMode_Post);
 	UnhookEvent("player_hurt", Event_PlayerHurt, EventHookMode_Pre);
 	UnhookEvent("player_team", Event_PlayerTeam, EventHookMode_Pre);
 	UnhookEvent("teamplay_round_start", Event_RoundStart, EventHookMode_Post);
 	UnhookEvent("teamplay_win_panel", Event_WinPanel, EventHookMode_Post);
-
+	
 	RemoveNormalSoundHook(sound_hook);
 	
-	for(new arena_index = 1; arena_index < g_iArenaCount; arena_index++)
+	for (int arena_index = 1; arena_index < g_iArenaCount; arena_index++)
 	{
-		if(g_bTimerRunning[arena_index])
+		if (g_bTimerRunning[arena_index])
 		{
-			KillTimer(g_tKothTimer[arena_index]);
+			delete g_tKothTimer[arena_index];
```

Comment:
please make sure to check if delete == killtimer in this case

Reply:
![Discord_Cf1VsuqGig](https://user-images.githubusercontent.com/20617661/100559611-0676e000-3292-11eb-83b8-d9ec700fd515.png)
![Discord_JGbyydALHc](https://user-images.githubusercontent.com/20617661/100559617-0b3b9400-3292-11eb-9938-b8181aa59d9f.png)